__tests__/app.test.js

```diff
@@ -24,3 +24,21 @@ describe("GET/api/topics", () => {
       });
   });
 });
+
+describe("GET/api/article/:article_id", () => {
+  test("should return an article object", () => {
+    return request(app)
+      .get("/api/articles/4")
+      .expect(200)
+      .then(({ body }) => {
+        expect(typeof body.result).toBe("object");
+        expect(body.result).toHaveProperty("author", expect.any(String));
```

Comment:
we shouldn't be expecting just any string/value on the properties of the returned review object here, we 100% need to be checking that we get back the correct information for the review specified. this logic is great for when we are checking a large array of objects which we expect to have the same properties (but not values), but in this case we know exactly what the object we are getting back should look like